mesonbuild/wrap/wrap.py

```diff
@@ -228,6 +232,26 @@ def get_hg(self, p):
                 subprocess.check_call(['hg', 'checkout', revno],
                                       cwd=checkoutdir)
 
+    def get_svn(self, p):
+        checkoutdir = os.path.join(self.subdir_root, p.get('directory'))
+        revno = p.get('revision')
+        is_there = os.path.isdir(checkoutdir)
+        if is_there:
+            svn_info_output = subprocess.getoutput(' '.join(['svn', 'info', '--show-item', 'revision', checkoutdir]))
```

Comment:
__[flake8]__
* [F841] local variable 'svn_info_output' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
fixed by 50376cca37324594c4f7ff9007b50ae837b46be2